### PR TITLE
Fix/3543  consistent borders for notification wall

### PR DESCRIPTION
**Pull request by MariePrandstetter**
#3543

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/src/css/NotificationWall.styl [hunk 1]
```diff
@@ -16,7 +16,7 @@
   overflow hidden
   transition 0.5s
   &__header
-    border-bottom 1px solid grey
+    // border-bottom 1px solid grey
```
Comment (raphj) [inline frontend/src/css/NotificationWall.styl:19]:
```suggestion
```

File: frontend/src/css/NotificationWall.styl [hunk 2]
```diff
@@ -72,6 +73,10 @@
   &__highlight
     font-weight bold
 
+// INFO - MB - 20200204 - overrides ListItemWrapper styles because .content is used in many places
+.notification > .notification__list > .content
```
Comment (raphj) [inline frontend/src/css/NotificationWall.styl:77]: Wasn't it possible to rename `.content` to `.notificationWall__content` or something like this?

Comment (MariePrandstetter) [reply frontend/src/css/NotificationWall.styl:75]: `.content` is also used in the activity feed, it is part of the function `ListItemWrapper`, so it was breaking the design when I tried to change it

**Review by Skylsmoi — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
Tested on branch.

**Comment by MariePrandstetter** [comment]
fix(frontend): consistent borders for notification wall

ref: #3543

**Pull request merged**